atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsFactory.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.sweep.metrics;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.util.CurrentValueMetric;
+import com.palantir.atlasdb.util.MetricsManager;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+
+public class SweepMetricsFactory {
+    private final TaggedMetricRegistry metricRegistry = new MetricsManager().getTaggedRegistry();
+
+    SweepMetric createDefault(String name) {
+        return new SweepMetricsFactory.ListOfMetrics(
+                createMeter(name, UpdateEvent.ONE_ITERATION, false),
+                createHistogram(name, UpdateEvent.ONE_ITERATION, true),
+                createHistogram(name, UpdateEvent.FULL_TABLE, false));
```

Comment:
I think we use ExponentiallyDecayingReservoir for histograms. It's docs follow:

```
    /**
     * Creates a new {@link ExponentiallyDecayingReservoir} of 1028 elements, which offers a 99.9%
     * confidence level with a 5% margin of error assuming a normal distribution, and an alpha
     * factor of 0.015, which heavily biases the reservoir to the past 5 minutes of measurements.
     */
```

1. We saw iterations of sweep having an interval of 7s between each other. Do we want histograms for each iteration?
2. Since sweeping two full tables can take a while, do we also want histograms for `UpdateEvent.FULL_TABLE` kinds of event?